sc/social/like/interfaces.py

```diff
@@ -212,3 +214,29 @@ class ISocialLikeSettings(model.Schema):
         required=False,
         default='',
     )
+
+    model.fieldset(
+        'open_graph', label=u'Open Graph', fields=['fallback_image', 'image_scale'])
+
+    form.widget('fallback_image', NamedImageFieldWidget)
+    fallback_image = schema.ASCII(
+        title=_(u'Image fallback'),
+        description=_(
+            u'help_fallback_image',
+            default=u'Content that does not have an image will be displayed on the share.\n '
```

Comment:
this messages makes no sense; please review.

Reply:
ok